simmadome/src/GamePage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {useState} from 'react';
+import {GameState, useListener} from './GamesUtil';
+import './GamePage.css';
+import Game from './Game';
+
+function GamePage() {
+	let searchparams = new URLSearchParams(window.location.search);
```

Comment:
Replace this with something like react-router or similar

Reply:
oh I see you already are. although I think that has a more direct way of accessing the query, and also it might be worth replacing it with a proper route (/game/<id> or similar) entirely?

Reply:
after some googling apparently React-router does not parse the search params for you as of 4.x. I don't quite know what you mean by "proper route" - this is already at /game/, the search params simply pass the id of the game to be watched

Reply:
oh, as in having the route be `/game/<id>` and taking the ID in the parameters there maybe?